### DISCUSSION TITLE
0.5.6

### DISCUSSION THREAD

**Discussion by z-huang** [z-huang/music#1356] [Announcements]
- Fix frequent random crash
- Fix crash when searching lyrics
- Upgrade Material 3
- Automatically scroll to the currently playing song in the queue

<hr /><em>This discussion was created from the release <a href='https://github.com/z-huang/InnerTune/releases/tag/v0.5.6'>0.5.6</a>.</em>

**Comment by Malopieds** [comment]
Hi, is this normal those things with the sliders/LinearProgress? Since Material 3 upgrade.

![image](https://github.com/user-attachments/assets/db9f7e64-cade-4496-a439-f01464e633bd)
Mini Player
![image](https://github.com/user-attachments/assets/cfed3d7d-3ef2-4e05-8371-2ac564c741c5)
Slider on Player
![image](https://github.com/user-attachments/assets/13540837-cc87-4bf3-a0a8-4fb8f1763487)
Sleep timer

**Reply by z-huang** [reply]
Yes, Material 3 changed the spec on sliders and linear indicator in Dec, 2023.

**Reply by Malopieds** [reply]
Oh yes, the m3 website was updated but not the android page. Thanks!

**Reply by mon-jai** [reply]
@z-huang Is that the intended component for the progress bar? Android 15's media controls look the other way.

![Screenshot_20240815-033919](https://github.com/user-attachments/assets/e91885fc-2d18-486b-9676-c5a4915750df)

**Comment by frutejuise** [comment]
When I click on a song, the mini player is black at first but changes to the bottom bar's color a split second later. My phone is a bit slow so that might be part of it..
The previous screenshot shows it, then it changes, but only when there wasn't a song before.
![Screenshot_20240812-124626~2.png](https://github.com/user-attachments/assets/7895bda2-1624-434d-9213-4bfd2a6522bd)

**Reply by frutejuise** [reply]
Update:
It also happens when dismissing the full sized player:

![Screenshot_20240812-125336~2.png](https://github.com/user-attachments/assets/db5841c5-b634-4fed-9304-09282dfbfe6c)

![Screenshot_20240812-125346.png](https://github.com/user-attachments/assets/d262d388-f048-4c61-aca9-ab168d2df628)

![Screenshot_20240812-125331~2.png](https://github.com/user-attachments/assets/61fd04e4-ae4a-417a-9374-7a25e03b0ac2)

**Reply by z-huang** [reply]
You turned on "Pure black", right? This is to make the player screen totally black, but this is the side effect. Create an issue for that, thanks.

**Reply by frutejuise** [reply]
Yes

**Comment by Anonuser-code** [comment]
Hi, when i click one of the 'mood and genres', it crash instantly.
![IMG_20240812_204156](https://github.com/user-attachments/assets/72353548-15bc-4ff7-8053-0d49aa33d3bc)

**Reply by frutejuise** [reply]
Yes

**Reply by GuiSousa135** [reply]
here is crashing too

**Reply by z-huang** [reply]
This may be another bug. Please create an issue and provide logs.

**Reply by z-huang** [reply]
May be fixed now.

**Reply by frutejuise** [reply]
Fixed in the latest debug apk 👍

**Comment by Anonuser-code** [comment]
Hi, when searching for the lyrics manually, it crashes the app completely.

**Reply by z-huang** [reply]
Please create an issue and provide logs.

**Reply by Anonuser-code** [reply]
> Please create an issue and provide logs.

How?

**Reply by z-huang** [reply]
https://developer.android.com/tools/logcat

**Reply by Anonuser-code** [reply]
> https://developer.android.com/tools/logcat

I did not understand anything. You just fix it. Even you can try the app you created and see the issue.

**Reply by z-huang** [reply]
I don't have this issue. That's why I need your logs so that I can know what's happening.

**Reply by Kiidandii** [reply]
How do I send you the logs. 
I am abit new to this

**Reply by Anonuser-code** [reply]
> I don't have this issue. That's why I need your logs so that I can know what's happening.

Simply describe how to send the logs. I don't understand anything from the link you sent, it says adb or something. Tell me how to do like you telling to an idiot.

**Reply by frutejuise** [reply]
You're getting this mad over not knowing how to do it when you don't have to?

**Reply by Anonuser-code** [reply]
> You're getting this mad over not knowing how to do it when you don't have to?

I'm not mad?

**Reply by frutejuise** [reply]
You called the dev an idiot.

**Comment by NEVIN-MJN** [comment]
Can you fix the back up and restore section if we give backup it just create backup file without data

**Comment by plpawel** [comment]
When i add album to favourites it doesnt show up in android auto and would it be possible to add random order from artists in android auto?

**Comment by Kiidandii** [comment]
Is there a way the quality of the music can be improved. 
I am always hearing a lot of distortion.

**Comment by paula879** [comment]
I'm having an issue with the music equalizer. It closes in between songs and i have to return to the app to re activate it.

**Comment by plpawel** [comment]
In android auto on album list below album name should be artist name? but there is only id
![20240814_0807511](https://github.com/user-attachments/assets/cab5c6f6-b13b-494a-b8a5-602263840c91)

**Reply by z-huang** [reply]
Fixed in d356c12325b88adfc5d0eff05b45b753aaa03e02

**Comment by mon-jai** [comment]
A dumb question: what is the difference between foss build and full build?

**Comment by Roxo7777** [comment]
Undownloaded songs are suddenly not playing (I can't debug at the moment;

![Screenshot_20240815_121446_InnerTune.jpg](https://github.com/user-attachments/assets/0ef620f9-a69b-4fea-89a8-e57b5eecf16b)

**Reply by Roxo7777** [reply]
The translator expressed it badly, I want to say that, suddenly, the songs that have not been downloaded are not even starting playing, and I can't download them either
